### PR TITLE
Handled Navbar problem

### PR DESCRIPTION
**Pull request by Gautam-Arora24**
I have handled the problem where Signout is even showing after the user gets loggedout. Kindly have a look.

### CODE REVIEW

**Comment by Gautam-Arora24** [comment]
Do lemme know if there is any problem

**Comment by Aman-Codes** [comment]
@Gautam-Arora24  Move the state isLoggedOut from HeaderComponent.js to Routes.js file so that it can be passed to other components as props

File: frontend/src/components/HeaderComponent.js [hunk 1]
```diff
@@ -1,70 +1,87 @@
...
-                </div>
-                </Navbar>
-            </React.Fragment>
-        )    
-}
-export default Header
\ No newline at end of file
+const Header = () => {
+  const [isNavOpen, setIsNavOpen] = useState(false);
+  const [isLoggedOut, setisLoggedOut] = useState(false);
```
Comment (Aman-Codes) [inline frontend/src/components/HeaderComponent.js:18]: Move this state to Routes.js file and pass it here as a prop

**Review by Aman-Codes — CHANGES_REQUESTED** [review]

File: frontend/src/components/HeaderComponent.js (hunk 1 above)
Comment (Aman-Codes) [reply frontend/src/components/HeaderComponent.js:18]: You can even use redux if you comfortable with it for maintaining isLoggedOut as a global state

**Comment by Gautam-Arora24** [comment]
I think using global state for just Signout is not worth it

**Comment by Gautam-Arora24** [comment]
Although I can pass it as props

**Comment by Aman-Codes** [comment]
Maybe you are right as we already do have an isAuth helper function to check the user logged in status and have private routes to ensure access to authenticated users only. So we don't need to maintain an extra global state for that.

**Comment by Aman-Codes** [comment]
I am reverting the merging of pull request as the problem still persist. Make a new pull request with updated changes.

**Pull request merged**